api/src/test/java/org/openmrs/DrugOrderTest.java

```diff
@@ -67,7 +67,7 @@ public void copy_shouldCopyAllDrugOrderFields() throws Exception {
 		drug.setConcept(new Concept());
 		drugOrder.setDrug(drug);
 		
-		OrderTest.assertThatAllFieldsAreCopied(drugOrder, null);
+		OrderTest.assertThatAllFieldsAreCopied(drugOrder, null, "orderGroup", "sortWeight");
```

Comment:
This is not correct, orderGroup and sortWeight need to be copied too, this should stay unchanged instead you update Order.copy to copy over these fields too